tests/integration/cmd_project_test.go

```diff
@@ -0,0 +1,53 @@
+package integration
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/odo/tests/helper"
+)
+
+var _ = Describe("odo project command tests", func() {
+	var project string
+	var context string
+
+	// This is run after every Spec (It)
+	var _ = BeforeEach(func() {
+		SetDefaultEventuallyTimeout(10 * time.Minute)
+		SetDefaultConsistentlyDuration(30 * time.Second)
+		context = helper.CreateNewContext()
+		os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "config.yaml"))
+		project = helper.CreateRandProject()
+	})
+
+	// Clean up after the test
+	// This is run after every Spec (It)
+	var _ = AfterEach(func() {
+		helper.DeleteProject(project)
+		helper.DeleteDir(context)
+		os.Unsetenv("GLOBALODOCONFIG")
+	})
+
+	Context("when running help for project command", func() {
+		It("should display the help", func() {
+			appHelp := helper.CmdShouldPass("odo", "project", "-h")
+			Expect(appHelp).To(ContainSubstring("Perform project operations"))
+		})
+	})
+
+	Context("when running project command app parameter in directory that doesn't contain .odo config directory", func() {
+		It("should successfully execute list along with machine readable output", func() {
+			helper.CmdRunner("odo", "project", "list")
+			time.Sleep(1 * time.Second)
```

Comment:
@mik-dass you missed it to remove i guess

Reply:
Because of #2117, we need to either comment out the test or fix the issue first

Reply:
I don't like this. We should never fix issues with sleep. But I guess it is kind of temporary ok :-( as long as we track it in issue

Reply:
I want to remove this and comment out the test for now or should we keep this?